packages/google_maps_flutter/google_maps_flutter_web/lib/src/convert.dart

```diff
@@ -264,16 +264,17 @@ gmaps.MarkerOptions _markerOptionsFromMarker(
 }
 
 gmaps.CircleOptions _circleOptionsFromCircle(Circle circle) {
-  final populationOptions = gmaps.CircleOptions()
+  final circleOptions = gmaps.CircleOptions()
     ..strokeColor = _getCssColor(circle.strokeColor)
     ..strokeOpacity = _getCssOpacity(circle.strokeColor)
     ..strokeWeight = circle.strokeWidth
     ..fillColor = _getCssColor(circle.fillColor)
     ..fillOpacity = _getCssOpacity(circle.fillColor)
     ..center = gmaps.LatLng(circle.center.latitude, circle.center.longitude)
     ..radius = circle.radius
-    ..visible = circle.visible;
-  return populationOptions;
+    ..visible = circle.visible
+    ..zIndex = circle.zIndex;
+  return circleOptions;
 }
```

Comment:
Out of curiosity, where aren't there unit tests for this file? Currently the unit test dir just point to integration tests, but a utility file like this one should be trivially unit testable in a normal Dart test context, no?

Reply:
We can't have VM unit tests on this file because 99% of it is converting between "Plugin Entities" <-> "`gmaps` Entities" (for the JS Interop, which uses `dart:html` extensively).

There's a few minor functions that could be extracted to VM tests, but anything that touches the `gmaps` namespace is tainted.

We could still write a convert_test.dart that unit-tested each of those methods in better isolation (but still as a chromedriver integration test), I just never got the time to dive that deep (most of these methods have a single entrypoint, and *that* is going to be have tests for sure, however, I wish I could prove this with some coverage metrics :P).

Reply:
> We can't have VM unit tests on this file because 99% of it is converting between "Plugin Entities" <-> "`gmaps` Entities" (for the JS Interop, which uses `dart:html` extensively).

Ah, right. I should have remembered that those objects were not from this plugin.